src/Microsoft.Health.Dicom.Core/Features/Query/QueryParser.cs

```diff
@@ -25,106 +23,85 @@ public partial class QueryParser : IQueryParser
         private readonly IDicomTagParser _dicomTagPathParser;
```

Comment:
`IDicomTagParser` could be make as static, and remove supporting sequence dicom tag  to further simplify

Reply:
I can do this in a separated PR